zerver/openapi/zulip.yaml

```diff
@@ -4521,6 +4521,18 @@ paths:
 
                                         **Changes**: In Zulip 9.0 (feature level 240), renamed
                                         'signup_notifications_stream_id' to `signup_announcements_stream_id`.
+                                    zulip_update_announcements_stream_id:
+                                      type: integer
+                                      description: |
+                                        The ID of the stream to which automated messages announcing
+                                        new features, new keyboard shortcuts, etc are sent.
```

Comment:
"new features or other end-user updates about the Zulip software." is probably a better description.

Reply:
Updated.